packages/mdc-textfield/foundation.js

```diff
@@ -159,7 +160,10 @@ class MDCTextFieldFoundation extends MDCFoundation {
     if (!this.outline_ || !this.label_) {
       return;
     }
-    const labelWidth = this.label_.getFloatingWidth();
+
+    const isDense = this.adapter_.hasClass(cssClasses.DENSE);
+    const labelScale = isDense ? 0.923 : 0.75;
```

Comment:
Should we create a `numbers` object in `constants.js` for these?

Reply:
Good idea. Done!